cpp/oneapi/dal/algo/jaccard/detail/vertex_similarity_ops.hpp

```diff
@@ -23,21 +23,22 @@ namespace oneapi::dal::preview {
 namespace jaccard {
 namespace detail {
 
-template <typename... Options>
+template <typename Float, class Method, typename Graph>
 struct vertex_similarity_ops_dispatcher {
-    similarity_result operator()(const descriptor_base &, const similarity_input &) const;
+    similarity_result operator()(const descriptor_base &descriptor,
```

Comment:
I see two styles of using '&', one - datatype &x, another datatype& y. What is the rule?